### PR TITLE
Update to AEM Groovy Console 18.0.0

### LINKED ISSUE #197: Provide package without groovy console embedded
I would like to start forking the ICF Next Groovyconsole as to be able to make improvements to it. AECU embeds the Groovyconsole it it's cloud package, making it hard to do this. 

Are you open to creating a package with and a package without the groovy console embedded? I don't mind contributing it myself but rather not do the effort if you aren't open to it :)

### PR DESCRIPTION
**Pull request by bdhoine**
Update to https://github.com/orbinson/aem-groovy-console/releases/tag/18.0.0

Only tested on AEM 6.5.10+ and AEMaaCS SDK 2022.11

Resolves #197

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -107,7 +107,7 @@ You can download the package from [Maven Central](https://repo1.maven.org/maven2
 
 ## Older AEM Versions (<6.5/Cloud)
 
-You can download the package from [Maven Central](https://repo1.maven.org/maven2/de/valtech/aecu/aecu.ui.apps/) or our [releases section](https://github.com/valtech/aem-easy-content-upgrade/releases). The aecu.ui.apps package will install the AECU software. It requires that you installed [Groovy Console](https://github.com/OlsonDigital/aem-groovy-console) before.
+You can download the package from [Maven Central](https://repo1.maven.org/maven2/de/valtech/aecu/aecu.ui.apps/) or our [releases section](https://github.com/valtech/aem-easy-content-upgrade/releases). The aecu.ui.apps package will install the AECU software. It requires that you installed [Groovy Console](https://github.com/orbinson/aem-groovy-console) before.
```
Comment (gruberrolandvaltech) [inline Readme.md:110]: This change can be reverted as for <6.5 the old Groovy Console needs to be used.

File: pom.xml [hunk 2]
```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
```
Comment (gruberrolandvaltech) [inline pom.xml:2]: Please fix merge conflict

File: ui.apps.groovyconsole/pom.xml [hunk 3]
```diff
@@ -1,75 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
```
Comment (gruberrolandvaltech) [inline ui.apps.groovyconsole/pom.xml:1]: Please fix merge conflict

**Review by gruberrolandvaltech — CHANGES_REQUESTED** [review]
Thanks a lot for taking over the maintenance of Groovy Console. There are just a few minor things and then we can merge it to our repo for further testing.

File: Readme.md (hunk 1 above)
Comment (bdhoine) [reply Readme.md]: yes, good point, reverted

**Review by gruberrolandvaltech — APPROVED** [review]

**Pull request merged**